apiserver/facades/controller/instancepoller/merge.go

```diff
@@ -102,15 +100,21 @@ func (o *mergeMachineLinkLayerOp) processExistingDevice(dev networkingcommon.Lin
 		}
 	}
 
+	// Collect normalised addresses for the incoming device.
+	// TODO (manadart 2020-07-15): We also need to set shadow addresses.
+	// These are sent where appropriate by the provider,
+	// but we do not yet process them.
+	incomingAddrs := o.MatchingIncomingAddrs(dev)
```

Comment:
Is implementing this functionality one of the next steps?

Reply:
It's not among the top items on my list - eventually.